soapui/src/main/java/com/eviware/soapui/analytics/providers/BaseAnalyticsProvider.java

```diff
@@ -60,6 +58,26 @@ public final String getUserCountry() {
         return System.getProperty("user.country", "n/a");
     }
 
+    String getInstallId() {
+        String hostName = "Unknown";
+        try {
+            hostName = InetAddress.getLocalHost().getHostName();
+        } catch (Exception ignore) {
+
+        }
+        try {
+            String installIdString = hostName + System.getProperty("user.name");
```

Comment:
Doesn't matter - since we don't want the real value but a way to identify the installation.